lms/djangoapps/teams/toggles.py

```diff
@@ -0,0 +1,38 @@
+"""
+Togglable settings for Teams behavior
```

Comment:
I know file rename is imperfect in git, but not sure if this was handled like a rename or not and if github just isn't picking it up.  Thanks.

Reply:
Good catch! It's really strange that this isn't a move, I'm pretty sure I did a `git mv` and not a simple `rm` or `git rm`.